### ISSUE TITLE
Finetune on IMDB and SST downsteam task and the performance is not good.

### ISSUE THREAD

**Issue by JinmingZhao** [thu-coai/SentiLARE#7]
Hi,

I have downloaded the pretrained model and the preprocessed data(*_newpos.txt) from google drive.
For SST the reported  val result is 55.04% in Table 15 and i get the performance 56.04%. however i only can get 84.92% on IMDB but the reported performance if 95.96% in Table 15.
And when i finetune the released model for downstream emotion recognition tasks, the results is poor than Bert-based model and RoBerta-Base model.

Is there any suggestions of these problems?  Thanks!

`for lr in 2e-5
do
    data_dir=/data7/emobert/resources/pretrained/sentilare/raw_data/sent/sst/
    CUDA_VISIBLE_DEVICES=${gpuid} python run_sent_sentilr_roberta.py \
            --cvNo 1 \
            --data_dir  ${data_dir} \
            --model_type roberta \
            --model_name_or_path ${pretrain_model_dir} \
            --task_name sst \
            --do_train \
            --do_eval \
            --max_seq_length 50 \
            --per_gpu_train_batch_size 32 \
            --per_gpu_eval_batch_size 32 \
            --learning_rate ${lr} \
            --num_train_epochs 8 \
            --warmup_steps 100 \
            --eval_all_checkpoints \
            --overwrite_output_dir \
            --seed 42 \
            --output_dir ${output_dir}/${corpus_name}_roberta_base_finetune_lr${lr}_bs32_m2
done`

**Comment by JinmingZhao** [comment]
When modidy the max_seq_length to 256 for IMDB, the perfermance is OK.
Maybe this model is not suitable for emotion(happy, sad, angry, neutral and so on) recognition tasks.

**Comment by JinmingZhao** [comment]
@kepei1106  If you have any suggestions for emotion recognition tasks based on this sentiment task-pretrained model, please help me.

**Comment by kepei1106** [comment]
Hi, sorry for the late reply. I haven't tried to fine-tune our model on the emotion recognition datasets yet. I guess the domain gap between the pre-training corpus (mainly including reviews) and the downstream datasets for emotion recognition may degrade the model performance.

Also, I wonder if you use the pre-processing codes we provide to enhance each word in your own datasets with its linguistic knowledge, including POS tags and word-level sentiment polarities. If not, you can refer to our pre-processing codes and the raw data / pre-processed data. This is important for the performance of SentiLARE.
